front-end/src/App.js

```diff
@@ -1,26 +1,16 @@
-import "./App.css";
-import RegisterCourse from "./components/RegisterCourse";
-import Login from "./components/Login";
-import CourseList from "./components/CourseList";
-import CourseDetail from "./components/CourseDetail";
-import { BrowserRouter, Routes, Route } from "react-router-dom";
-import Layout from "./components/Layout";
-import LandingPage from "./components/LandingPage";
-import UserInformationScreen from "./components/UserInformationScreen";
+import './App.css';
```

Comment:
This file does not need to be modified. The browser, router component and everything inside of it was added because of another story. You should be able to revert your changes and just let the browser router handle navigation to the register course page